cachito/web/api_v1.py

```diff
@@ -588,3 +588,40 @@ def get_request_logs(request_id):
     return flask.Response(
         stream_with_context(generate_stream_response(log_file_path)), mimetype="text/plain"
     )
+
+
+@api_v1.route("/requests/content-manifest", methods=["GET"])
+def get_content_manifest_by_requests():
+    """
+    Retrieve the content manifest associated with the given requests.
+
+    :return: a Flask JSON response
+    :rtype: flask.Response
+    :raise NotFound: if one of the given request is not found
+    :raise ValidationError: if any of the given request is not in the "complete" or "stale" state
+    """
+    arg = flask.request.args.get("requests")
+    if not arg:
+        return flask.jsonify([])
```

Comment:
This should be an empty content manifest, i.e. the return value of `ContentManifest.generate_icm(image_contents=None)`